libs/core/src/components/pds-textarea/pds-textarea.scss

```diff
@@ -1,38 +1,35 @@
-  --color-error: var(--pine-color-red-300);
-  --color-placeholder: var(--pine-color-neutral-charcoal-200);
-  --color-disabled: var(--pine-color-neutral-charcoal-200);
-
-  --border-color-default: var(--pine-color-neutral-grey-400);
-  --border-color-hover: var(--pine-color-neutral-grey-500);
-  --border-color-focus-visible: var(--pine-color-neutral-grey-300);
-  --border-color-error: var(--pine-color-red-300);
-
-  --box-shadow-focus: 0 0 0 2px var(--focus-visible-outline-color);
-  --box-shadow-focus-error: 0 0 0 2px var(--focus-visible-outline-color-error);
-
-  --focus-visible-outline-color: var(--pine-color-primary-200);
-  --focus-visible-outline-color-error: var(--pine-color-red-200);
-
-  --label-font-size: var(--pine-font-size-body);
-  --label-font-weight: var(--pine-font-weight-medium);
-  --label-line-height: var(--pine-line-height-md);
-  --label-margin-block-end: var(--pine-spacing-xs);
-
-  --field-font-size: var(--pine-font-size-body);
-  --field-font-weight: var(--pine-font-weight-regular);
-  --field-line-height: var(--pine-line-height-md);
-  --field-padding-block: var(--pine-spacing-xs);
-  --field-padding-inline: var(--pine-spacing-sm);
-
-  --helper-message-font-size: var(--pine-font-size-body-xs);
-  --helper-message-font-weight: var(--pine-font-weight-regular);
-  --helper-message-line-height: var(--pine-line-height-sm);
-  --helper-message-margin-block-start: var(--pine-spacing-xs);
+  --box-shadow-focus: 0 0 0 2px var(--color-focus-visible-outline);
+  --box-shadow-focus-error: 0 0 0 2px var(--color-focus-visible-outline-error);
+
+  --color: var(--pine-color-charcoal-400);
+  --color-background: var(--pine-color-white);
+  --color-background-disabled: var(--pine-color-grey-200);
+  --color-border-default: var(--pine-color-grey-400);
+  --color-border-error: var(--pine-color-red-300);
+  --color-border-focus-visible: var(--pine-color-grey-300);
+  --color-border-hover: var(--pine-color-grey-500);
+  --color-disabled-default: var(--pine-color-charcoal-200);
+  --color-error-default: var(--pine-color-red-300);
+  --color-focus-visible-outline: var(--pine-color-primary-200);
+  --color-focus-visible-outline-error: var(--pine-color-red-200);
+  --color-placeholder-default: var(--pine-color-charcoal-200);
+
+  --font-size-field: var(--pine-font-size-100);
+  --font-size-helper-message: var(--pine-font-size-075);
+  --font-size-label: var(--pine-font-size-100);
+  --font-weight-field: var(--pine-font-weight-normal);
+  --font-weight-helper-message: var(--pine-font-weight-normal);
+  --font-weight-label: var(--pine-font-weight-medium);
+
+  --line-height-field: var(--pine-line-height-150);
+  --line-height-helper-message: var(--pine-line-height-125);
+  --line-height-label: var(--pine-line-height-150);
+
+  --spacing-margin-block-end-label: var(--pine-spacing-100);
+  --spacing-margin-block-start-helper-message: var(--pine-spacing-100);
+  --spacing-padding-block-field: var(--pine-spacing-100);
```

Comment:
Not a blocker and shouldn't hold up the PR at all, but these are causing the most confusion for me. Seems to me if they all have the same value then they could be combined to prevent so many one-off variables. I'm just not entirely clear on how this is better than just putting the values in the selectors.